internal/publish/publish.go

```diff
@@ -332,6 +332,7 @@ func (h *PublishHandler) process(ctx context.Context, data *verifyapi.Publish, b
 			pubResponse: &verifyapi.PublishResponse{
 				ErrorMessage: message,
 				Code:         verifyapi.ErrorBadRequest,
+				Warnings:     transformWarnings,
```

Comment:
this only sets transformWarnings if there is also an error

Reply:
line 419 needs to also set warnings if it isn't empty, right?

Reply:
Ah and further down too. Updated.